packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -498,50 +495,68 @@ contract GuardiansRegistry is IGuardiansRegistry, ControlledRecoverable, Control
     }
 
     /**
-    * @dev Tell the total amount of active guardian tokens at the given term id
-    * @param _termId Term ID querying the total active balance for
+    * @dev Tell the total amount of active guardian tokens for a given term id
+    * @param _termId Term ID to query on
     * @return Total amount of active guardian tokens at the given term id
     */
     function totalActiveBalanceAt(uint64 _termId) external view returns (uint256) {
         return _totalActiveBalanceAt(_termId);
     }
 
     /**
-    * @dev ERC900 - Tell the total amount of tokens of guardian. This includes the active balance, the available
-    *      balances, and the pending balance for deactivation. Note that we don't have to include the locked
-    *      balances since these represent the amount of active tokens that are locked for drafts, i.e. these
-    *      are included in the active balance of the guardian.
-    * @param _guardian Address of the guardian querying the total amount of tokens staked of
+    * @dev Tell the total balance of tokens held by a guardian
+    *      This includes the active balance, the available balances, and the pending balance for deactivation.
+    *      Note that we don't have to include the locked balances since these represent the amount of active tokens
+    *      that are locked for drafts, i.e. these are already included in the active balance of the guardian.
+    * @param _guardian Address of the guardian querying the balance of
     * @return Total amount of tokens of a guardian
     */
-    function totalStakedFor(address _guardian) external view returns (uint256) {
-        return _totalStakedFor(_guardian);
+    function balanceOf(address _guardian) external view returns (uint256) {
+        return _balanceOf(_guardian);
     }
 
     /**
-    * @dev Tell the balance information of a guardian
-    * @param _guardian Address of the guardian querying the balance information of
+    * @dev Tell the total balance of tokens held by a guardian for a given term id
+    *      This includes the active balance, the available balances, and the pending balance for deactivation.
+    *      Note that we don't have to include the locked balances since these represent the amount of active tokens
+    *      that are locked for drafts, i.e. these are already included in the active balance of the guardian.
+    * @param _guardian Address of the guardian querying the balance of
+    * @param _termId Term ID to query on
+    * @return Total amount of tokens of a guardian
+    */
+    function balanceOfAt(address _guardian, uint64 _termId) external view returns (uint256) {
```

Comment:
⚠️ I wasn't quite sure about:

1. Where to best test this
2. Whether the logic here is correct. It mimics how `_balanceOf()` works.

Reply:
1. I'd say in `guardians-registry-activation` since you will need to query the active balance for different terms
2. I'd it mimics `_balanceOf` but for a given `_termId` right?

Reply:
> I'd it mimics _balanceOf but for a given _termId right?

Yes, that was the intention (since we had the `at()` versions for the detailed and other balance getters).

Reply:
Removed in a8f7fa7, as we clarified that the only checkpointed balances we maintain are the active ones.